lectures/5-Tutorials/SymmetricTensors.md

```diff
@@ -0,0 +1,793 @@
+
+This is a one-dimensional system defined as a spin chain, where each physical 'spin' corresponds to a so-called [Fibonacci anyon](https://arxiv.org/abs/0902.3275). There are two such Fibonacci anyons, which we will denote as $1$ and $\tau$. They obey the fusion rules
+```{math}
+1 \otimes 1 = 1, \quad 1 \otimes \tau = \tau, \quad \tau \otimes \tau = 1 \oplus \tau.
+```
+The Hilbert space of a chain of Fibonacci anyons is not a regular tensor product space, but rather a *constrained Hilbert space* where the only allowed basis states are labeled by valid Fibonacci fusion configurations.
+In the golden chain model, we define a nearest-neighbor Hamiltonian on this Hilbert space by imposing an energy penalty when any two neighboring anyons fuse to a $\tau$ anyon.
+
+Even just writing down an explicit expression for this interaction on such a constrained Hilbert space is not entirely straightforward. However, using the framework of symmetric tensors it can actually be explicitly constructed in a very straightforward way. Indeed, TensorKit.jl supports a dedicated [`FibonacciAnyon`](https://jutho.github.io/TensorKit.jl/latest/lib/sectors/#TensorKit.FibonacciAnyon) sector type which can be used to construct precisely such a constrained Fibonacci-graded vector space. A Hamiltonian
+```{math}
+H = \sum_{\langle i,j \rangle} h_{ij}
+```
+which favors neighboring anyons fusing to the vacuum can be constructed as a tensor map on the product space of two Fibonacci-graded physical spaces
+```{code-cell} julia
+:tags: [hide-output]
+V = Vect[FibonacciAnyon](:τ => 1)
+```
+and assigning the following nonzero block value to the two-site fusion trees
+```{figure} ../_static/SymmetricTensors/Fib_fusiontrees.svg
+:scale: 12%
+:name: Fib_fusiontrees
+```
+This allows us to define this, at first sight, exotic and complicated Hamiltonian in a few simple lines of code,
+```{code-cell} julia
+h = TensorMap(ones, V ⊗ V ← V ⊗ V)
+for (s, f) in fusiontrees(h)
+    h[s, f] .= f.coupled == FibonacciAnyon(:I) ? -1 : 0
+end
+h
+```
+
+```{note}
+In the previous section we have stressed the role of Clebsch-Gordan coefficients in the structure of symmetric tensors, and how they can be used to map between the representation of an operator in the irrep basis and its symmetric tensor representation. However, for categorical symmetries such as the Fibonacci anyons, there are no Clebsch-Gordan coefficients. Therefore, the 'matrix elements of the operator in the irrep basis' are not well-defined, meaning that a Fibonacci-symmetric tensor cannot actually be converted to a plain array in a meaningful way.
+```
+
+
+## Closing Remarks
+
+TODO
+
+- Mention something about product symmetries.
```

Comment:
Hubbard would be cool, but I think we can keep this as a todo for now.

classical stat-mech things with explicit fourier expansion and spherical harmonics are also cool, but maybe also ok as todo for now